WcaOnRails/app/assets/javascripts/application.js

```diff
@@ -366,4 +366,3 @@ $(function() {
     form.submit();
   });
 });
-cookiesEu.init()
```

Comment:
Is this change intentional?

Reply:
Oh yes. I don't know why we put this here in the first place, because [`cookiesEu` automatically invokes it's `init` function when the page loads](https://github.com/infinum/cookies_eu/blob/3a0ad36b51b4be0b038c41c89912ea301ecfb90a/app/assets/javascripts/cookies_eu.js#L52). In fact, this `init` method is totally undocumented, so I don't think there's any guarantee that they keep it around, as it's just an implementation detail.